annotations/build.gradle

```diff
@@ -1,5 +1,4 @@
 apply plugin: 'java'
-apply plugin: 'com.novoda.bintray-release'
```

Comment:
Is `apply plugin: 'com.github.dcendents.android-maven'` not needed?
Others `build.gradle` has the line.

Reply:
This module and `processor` module is not android project.
So, we don't need this plugin.

Reply:
I just understand, thanks!

Reply:
👍